### PR TITLE
LinkedIn Ads source documentation

### PR DESCRIPTION
**Pull request by mohan-h**
Initial commit

### CODE REVIEW

File: help/sources/tutorials/api/create/advertising/linkedin-ads.md
Comment (carlojavier) [inline help/sources/tutorials/api/create/advertising/linkedin-ads.md]: Can we add a brief definition or description to what `pivot` is?

File: help/sources/tutorials/api/create/advertising/linkedin-ads.md
Comment (carlojavier) [inline help/sources/tutorials/api/create/advertising/linkedin-ads.md]: - Let's rephrase the first sentence to: "...is a marketing solution by LinkedIn that can reach over 830..."
- And a slight restructuring to the second sentence: "...key insights on performance such as clicks, impressions, and ad spend, as well as demographic information like metrics by demographic values at the account, campaign, and creative levels."

File: help/sources/tutorials/api/create/advertising/linkedin-ads.md
Comment (carlojavier) [inline help/sources/tutorials/api/create/advertising/linkedin-ads.md]: - Some restructuring here: "...to retrieve your LinkedIn campaign data, along with all performance and metrics. You can then bring that data to Experience Platform, where you can execute analytics."
- "The data is returned from a specified date with a default of six month retention period for daily data." (Let me know if this rephrasing still works for the meaning.)

File: help/sources/tutorials/api/create/advertising/linkedin-ads.md
Comment (carlojavier) [inline help/sources/tutorials/api/create/advertising/linkedin-ads.md]: For the API side, can we wrap the credentials in backticks and write them out in camelcase

File: help/sources/tutorials/api/create/advertising/linkedin-ads.md
Comment (carlojavier) [inline help/sources/tutorials/api/create/advertising/linkedin-ads.md]: Let's move the second sentence into an actual note:

>[!NOTE]
>
>LinkedIn Ads only allows data retrieval from active campaigns.The data is returned from a specified date with a default of six month retention period for daily data."

File: help/sources/tutorials/ui/create/advertising/linkedin-ads.md
Comment (carlojavier) [inline help/sources/tutorials/ui/create/advertising/linkedin-ads.md]: Semi-colon (`;`) is unnecessary here

File: help/sources/tutorials/ui/create/advertising/linkedin-ads.md
Comment (carlojavier) [inline help/sources/tutorials/ui/create/advertising/linkedin-ads.md:122]: Can the screenshot be updated here? There is a typo on the dataset name "Tset123"

File: help/sources/tutorials/ui/create/advertising/linkedin-ads.md
Comment (carlojavier) [inline help/sources/tutorials/ui/create/advertising/linkedin-ads.md]: Same as API:
Let's rephrase the first sentence to: "...is a marketing solution by LinkedIn that can reach over 830..."
And a slight restructuring to the second sentence: "...key insights on performance such as clicks, impressions, and ad spend, as well as demographic information like metrics by demographic values at the account, campaign, and creative levels."

File: help/sources/tutorials/ui/create/advertising/linkedin-ads.md
Comment (carlojavier) [inline help/sources/tutorials/ui/create/advertising/linkedin-ads.md]: Same comment as API about moving the note into its own [!NOTE] tag

File: help/sources/tutorials/ui/create/advertising/linkedin-ads.md
Comment (carlojavier) [inline help/sources/tutorials/ui/create/advertising/linkedin-ads.md:104]: I think we need to preface this particular part about why a user needs to schedule specifically for Day or Hour? Can we add a brief intro here on why this is recommended/required?

File: help/sources/tutorials/ui/create/advertising/linkedin-ads.md
Comment (carlojavier) [inline help/sources/tutorials/ui/create/advertising/linkedin-ads.md]: Need to remove space between ">" and the text

File: help/sources/tutorials/ui/create/advertising/linkedin-ads.md
Comment (carlojavier) [inline help/sources/tutorials/ui/create/advertising/linkedin-ads.md]: let's wrap the entries in the fields column in backticks

**Review by carlojavier — CHANGES_REQUESTED** [review]
@mohan-h looking good, some comments added!

File: help/sources/tutorials/api/create/advertising/linkedin-ads.md
Comment (mohan-h) [reply help/sources/tutorials/api/create/advertising/linkedin-ads.md]: Used _The data is returned from a specified date within the six month default retention period for daily data._

Comment (mohan-h) [reply help/sources/tutorials/api/create/advertising/linkedin-ads.md]: Used _The data is returned from a specified date within the six month default retention period for daily data._

**Comment by carlojavier** [comment]
Closing due to inactivity. The PR is ready on internal, still just waiting on approval from internal on when it can be published.

**Pull request closed without merging**
